extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts

```diff
@@ -107,12 +107,13 @@ export class ChePluginServiceImpl implements ChePluginService {
         }
     }
 
-    private getAxiosInstance(): AxiosInstance {
-        if (!this.isItNode()) {
-            return axios;
-        }
-
+    /**
+     * URI for which an axios instance will be created.
```

Comment:
The returned axios instance isn't tied to a specific URI, so I would remove this sentence.

Reply:
Done